frontend/components/TableContainer/DataTable/InstallerActionCell/InstallerActionCell.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { InjectedRouter } from "react-router";
+
+import PATHS from "router/paths";
+import { getPathWithQueryParams } from "utilities/url";
+import { IFleetMaintainedApp } from "interfaces/software";
+
+import { softwareAlreadyAddedTipContent } from "pages/SoftwarePage/SoftwareAddPage/SoftwareFleetMaintained/FleetMaintainedAppDetailsPage/FleetAppDetailsForm/FleetAppDetailsForm";
+import classnames from "classnames";
+import Icon from "components/Icon";
+import Button from "components/buttons/Button";
+import TooltipWrapper from "components/TooltipWrapper";
+import TextCell from "../TextCell";
+
+const baseClass = "installer-action-cell";
+
+interface IInstallerActionCellProps {
+  value?: Omit<IFleetMaintainedApp, "name" | "version"> | null;
+  router?: InjectedRouter;
+  className?: string;
+  teamId?: number;
+}
+
+const InstallerActionCell = ({
+  value,
+  router,
+  className = "w250",
+  teamId,
+}: IInstallerActionCellProps) => {
+  const cellClasses = classnames(baseClass, className);
+
+  // Some FMAs are not supported for Windows
+  if (!value) {
+    return (
+      <TextCell
+        className={cellClasses}
+        emptyCellTooltipText="Currently unavailable for Windows."
+      />
+    );
+  }
```

Comment:
```suggestion
  // Not all FMAs are supported for all platforms
  if (!value) {
    return (
      <TextCell
        className={cellClasses}
        emptyCellTooltipText="Currently unavailable for this platform."
      />
    );
  }
```

I figure we'll be throwing a bunch of EXE installers into FMA sooner rather than later, so our WinFMA light might actually get bigger than our macOS one.